Makefile

```diff
@@ -472,8 +463,8 @@ endif
 ifneq ($(FALCON),)
 CC=m68k-atari-mint-gcc
 CXX=m68k-atari-mint-g++
-CFLAGS += -m68020-60 -DMSB_FIRST
-CXXFLAGS += -m68020-60 -DMSB_FIRST
+CFLAGS += -m68020-60
```

Comment:
I don't understand the  big endian change here, this -DMSB_FIRST is needed by mrboom.c, this will break both the SDL and libretro versions...

Reply:
retro_endianness.h now defines MSB_FIRST by itself based on built-in compiler defines